### ISSUE TITLE
Introduce "RadioButton" to handle RadioButton selection 

### ISSUE THREAD

**Issue by mpost** [eclipsesource/tabris-js#1688]
### Problem description

Currently the Composite handles all RadioButton selection and deselection with no particular API for that use case.

### Expected behavior

We should introduce a new `RadioGroup` that handles selection and deselection and allows to get/set/observe the selection.

**Comment by tbuschto** [comment]
`RadioGroup` would extend `Composite`, or would it be a non-widget? If it's a widget, would it also allow non radio button children? I would say no, then the selection can simply be represented by a `selectionIndex` property like `Picker`. Otherwise we could only have a `selection` property like `TabFolder`.

**Comment by cookieguru** [comment]
@tbuschto I think it would be convenient to have any type of children, for cases when you are reliant on the wrapping of a `TextView` to determine the vertical positioning of the next radio button.

**Comment by mpost** [comment]
I would agree with @cookieguru The `RadioGroup` should be a regular `Composite` with added functionality to better deal with `RadioButton`s.
